### DISCUSSION TITLE
Odd layer shift

### DISCUSSION THREAD

**Discussion by Thunder1723** [supermerill/SuperSlicer#2062] [Q&A]
I have a layer shift right below where the benchy starts its first overhang and its is most definitely on the slicers side because at vaguely similar cura settings I get no such shift when printing.  I even slowed down outside perimeters to slower than what I had in cura and while its reduced the amount of shift I still cannot figure out what is causing it.  Anyone know of other settings to fiddle with?

**Comment by supermerill** [comment]
if it's the bigger flow from the overhang causing it, increase the ovrehang flow threshold from 75%? to ~110%?
